### DISCUSSION TITLE
exec command with script in gotmpl file

### DISCUSSION THREAD

**Discussion by danilo-patrucco** [helmfile/helmfile#1374] [Questions] [answered]
Hello,

I am trying to run a script in the gotmpl file, using the following code

{{ exec "./myfolder/myscript.sh" (list "arg1) | indent 2 }} 

but it cannot find the script. I tried the full path and it still does not work. 

I don't even need the arg1, I just need to run the script, but I'm not sure what I am doing wrong.

I tried to use a simple command (pwd) and it worked

Thanks for your attention

Regards

**Comment by yxxhero** [comment]
@danilo-patrucco please  post the log by add `--debug` flag.

**Reply by danilo-patrucco** [reply, marked as answer]
Hi, Thank you for the answer, I added --debug and then searching some section of the debug error I was able to identify the issue.

I was calling the script with exec, but I should have taken a different approach, the following code solved the issue.

{{ exec  "sh" (list "/myfolder/myscript.sh") | indent 6 }}

By adding the sh in addition to the file location (full path) I was able to make it work.

Thank you so much for your help !
